docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -324,6 +324,35 @@ and needs to be manually advanced):
   links to the security registrar (http://www.silverstripe.org/download/security-releases)
   match the pages saved in draft.
 
+#### Basing a new release on a previous one (tweak releases)
+
+Commonly a stable release will need to mirror the contents of the release
+candidate that preceded it, sometimes with a small set of additional commits.
+However, running the standard `cow release` command will create a release that
+includes all the latest commits on the branches it targets, which can include
+unaudited code. A **tweak release** includes only the commits present in the
+previous tagged release by default, and can optionally include additional
+commits when necessary. To create one, use the `release:checkout-tagged-base`
+command:
+
+1. `cow release:create <new-version>` to create the new release.
+2. `cow release:plan <new-version>` to generate a plan for the new release.
+3. `cow release:checkout-tagged-base <new-version>` to shift all of the modules
+  to the commit in the release branch that matches the previous version tag.
+  * **How?** This command finds the last common commit between the previous
```

Comment:
`This command finds the last common commit` - is this always just going to be the commit the previous tag is against?

Reply:
For recipe modules (e.g. `silverstripe/installer`, `silverstripe/recipe-core`), tags always contain at least one commit that isn't in the branch, which sets exact tags on their dependencies. That commit never makes it back into the mainline branch.

An earlier revision of this PR included this detail here, but it seemed a little long-winded and not super relevant to the operation. It's also somewhat referenced below (in the section that tells you not to merge tags back into branches.)